src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.Desktop.cs

```diff
@@ -100,10 +101,25 @@ internal bool EnsureResolvedUnhooked()
             try
             {
                 var assemblyName = new AssemblyName(args.Name);
-                string? bestPath = GetBestPath(assemblyName);
-                if (bestPath is not null)
+                var isSatelliteAssembly =
+                    assemblyName.CultureInfo is CultureInfo cultureInfo &&
+                    !cultureInfo.IsNeutralCulture &&
```

Comment:
Why did you check for `IsNeutralCulture` here? That wasn't necessary in .NET Core. Did I miss a case?

Reply:
I think I was trying to be make sure neutral culture requests didn't try to load satellite assemblies, but I think it's best if I just use the same logic you did.